modules/gsp-cluster/monitoring-system.tf

```diff
@@ -42,10 +60,7 @@ resource "aws_iam_policy" "cloudwatch_log_shipping_policy" {
 
 resource "aws_iam_policy_attachment" "cloudwatch_log_shipping_policy" {
   name = "${var.cluster_name}_cloudwatch_log_shipping_role_policy_attachement"
-  roles = [
-    aws_iam_role.cloudwatch_log_shipping_role.name,
-    module.k8s-cluster.kiam-server-node-instance-role-name,
```

Comment:
I'm a little unsure about the removal of this - it's presumably to allow fluentd-cloudwatch to run on the cluster management nodes (aka kiam server nodes) directly, where it'll have access to the metadata endpoint. If it sees the metadata endpoint and ignores the web identity token in its environment, then this will break.

On the other hand I notice that in charts/gsp-cluster/values.yaml where we set fluentd-cloudwatch tolerations it only tolerates `node-role.kubernetes.io/master`, which... isn't a thing? it won't tolerate the cluster-management taint... Definitely feels like I've stumbled across an existing bug here.

Reply:
It looks like actually that value in values.yaml does not make it into the DaemonSet in our cluster, which just has:
```
      tolerations:
      - effect: NoSchedule
        operator: Exists
```
I wonder how this works on the CI nodes.

Reply:
Looks like the above value came from charts/gsp-cluster/values.yaml but it's overridden to the real value in the cluster in modules/gsp-cluster/data/values.yaml